workspaces/src/worker/impls.rs

```diff
@@ -200,48 +198,51 @@ where
     T: NetworkClient + Send + Sync + ?Sized,
 {
     /// Provides a list of changes in block associated with the given block reference.
-    pub async fn changes_in_block(
-        &self,
-        block_reference: BlockReference,
-    ) -> Result<RpcStateChangesInBlockByTypeResponse> {
-        self.client().changes_in_block(block_reference).await
+    pub fn changes(&self, account_ids: Vec<AccountId>) -> Query<'_, StateChanges> {
```

Comment:
The inputs to this should just be a slice instead of a `Vec` as to make it more ergonomic for users to make use of
```suggestion
    pub fn changes(&self, account_ids: &[AccountId]) -> Query<'_, StateChanges> {
```